olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -8,9 +8,9 @@ COPY --from=opm /bin/opm /bin/opm
 COPY configs /configs
 
 RUN /bin/opm init serverless-operator --default-channel=stable --output yaml >> /configs/index.yaml
-RUN /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.26.0:serverless-stop-bundle \
+RUN /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.27.0:serverless-stop-bundle \
```

Comment:
Perhaps, when we're changing 1.26.0 to 1.27.0 in this file we should also change 1.27.0 to 1.27.1 on this line: https://github.com/openshift-knative/serverless-operator/blob/2566f7641e913c8a917cc0fb3ed35d9598479a0b/olm-catalog/serverless-operator/index/Dockerfile#L12